examples/pubsub-ts/index.ts

```diff
@@ -48,7 +48,7 @@ const schema = new google.pubsub.v1.Schema("schema", {
         ],
     }),
     name: "my-schema",
-    type: "avro",
+    type: "AVRO",
```

Comment:
Should this be an enum?

Reply:
It is an enum, there are two ways to express them in TS: a string or a constant. I changed the other example to a const and this example is a string (but `"avro"` wouldn't compile).